internal/bgp/frr/config.go

```diff
@@ -23,19 +23,31 @@ var reloaderPidFileName = "/etc/frr_reloader/reloader.pid"
 // (https://pkg.go.dev/text/template#hdr-Nested_template_definitions), this
 // should also be considered.
 const configTemplate = `
-log stdout {{.Loglevel}}
+log file /etc/frr/frr.log {{.Loglevel}}
```

Comment:
```
k -n metallb-system logs speaker-qfr82 -c frr
Started watchfrr
```
its not generating any logs from the container

Reply:
but we'd like it to, right?

Reply:
I am not sure if its because of the way the frr process run probably like another TODO at the moment redirect to file was the only way to get debug logs for debugging